core/src/text/textBuffer.cpp

```diff
@@ -87,16 +87,23 @@ bool TextBuffer::addLabel(const std::string& _text, Label::Transform _transform,
     float inf = std::numeric_limits<float>::infinity();
     float x0 = inf, x1 = -inf, y0 = inf, y1 = -inf;
 
+    // Stroke width is normalized by the distance of the SDF spread, then scaled
+    // to a char, then packed into the "alpha" channel of stroke. The .25 scaling
+    // probably has to do with how the SDF is generated, but honestly I'm not sure
```

Comment:
Ok buckle up, here's my model of this:

The SDF for a glyph goes from 0 to 255 over a distance of `blurSpread` pixels. `strokeWidth` is in pixels, so to find its SDF threshold we divide by `blurSpread` and multiply by 255. Now recall that one half of the SDF value range is the interior of contours, so to make sure our stroke is measured from the outside of a contour we multiply by 0.5. 

This is the part where I start obsessively measuring stroke pixel widths in both tangram-js and tangram-es, because the strokes I was getting were twice the size that they were in the browser. After an extended discussion we discovered that tangram-js is probably producing half-sized strokes, so to match that here we add another factor of 0.5 and now the whole thing is scaled by 0.25.